main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -1,6 +1,18 @@
 {
   "methods": {
     "java.lang.Boolean::getBoolean": "Env, IO",
+    "java.lang.Character::toString": "IO",
+    "java.lang.Character::codePointAt": "IO",
```

Comment:
I suppose the reason that codePointAt has IO is because it reads from an array?

Reply:
And so it should probably have a heap effect in that array region, but we have no way to express that...

Reply:
We probably need to discuss this at the next meeting.

Reply:
Ah good point. It was mostly just because it can throw `IndexOutOfBounds`

Reply:
Roger. I think we are ignoring exceptions- at least for now.